tox.ini

```diff
@@ -30,6 +30,7 @@ deps =
   flake8-import-order==0.9.2
   pep8-naming
   teamcity-messages
+  pycodestyle==2.1.0
```

Comment:
👍  for pinning the versions.

In a separate PR, the other dependencies can be pinned to specific versions.